Snakefile

```diff
@@ -410,3 +417,76 @@ rule targetSafety:            # Process data from different sources that describ
             --output {output}
         opentargets_validator --schema {params.schema} {output}
         """
+# --- Evidence generation for PPP --- #
+rule ot_crispr:               # Generating evidence for OTAR CRISPR screens
```

Comment:
```suggestion

rule ot_crispr:               # Generating PPP evidence for OTAR CRISPR screens
```

This is just to bring the style in alignment with existing rules, & leave a blank line